build_deps/toolchains/gpu/cuda_configure.bzl

```diff
@@ -555,7 +555,7 @@ def _find_libs(repository_ctx, cuda_config):
             repository_ctx,
             cpu_value,
             cuda_config.config["cuda_library_dir"],
-            cuda_config.cuda_version,
```

Comment:
is cuda_version removed?

Reply:
The runtime version is what's needed for so naming.